app/utils/gallery/index.ts

```diff
@@ -105,10 +106,10 @@ export function measureItem(ref: React.RefObject<any>, sharedValues: GalleryMana
     try {
         const measurements = measure(ref);
 
-        sharedValues.x.value = measurements.pageX;
-        sharedValues.y.value = measurements.pageY;
-        sharedValues.width.value = measurements.width;
-        sharedValues.height.value = measurements.height;
+        sharedValues.x.value = measurements?.pageX || 999999;
+        sharedValues.y.value = measurements?.pageY || 999999;
+        sharedValues.width.value = measurements?.width || 0;
+        sharedValues.height.value = measurements?.height || 0;
```

Comment:
Curious about what these default values try to accomplish

Reply:
Set some values when the item could not be measured as is outside of the viewport